spec/features/collection_drafts/forms/validation_spec.rb

```diff
@@ -522,16 +516,17 @@
 
       within '#draft_spatial_extent_horizontal_spatial_domain_geometry_bounding_rectangles_0' do
         fill_in 'North', with: 'asdf'
-        fill_in 'West', with: ''
-        fill_in 'East', with: ''
-        fill_in 'South', with: ''
+        find('#draft_spatial_extent_horizontal_spatial_domain_geometry_bounding_rectangles_0_west_bounding_coordinate').click
+        find('#draft_spatial_extent_horizontal_spatial_domain_geometry_bounding_rectangles_0_east_bounding_coordinate').click
+        find('#draft_spatial_extent_horizontal_spatial_domain_geometry_bounding_rectangles_0_south_bounding_coordinate').click
```

Comment:
The prior way of doing it no longer works?

Reply:
Also as I mentioned, previously when filling in with an empty string that served to tell the page that the input field was visited and could trigger validation. Now that is not happening, so the click serves to 'visit' the field but not enter anything.